jboss-kie-wildfly-common/tests/bats/resources/standalone-openshift-logging.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0"?>
```

Comment:
you don't need to create a new file just for the assertions.
as it is just tests, its up to you to choose, but you could use the original standalone-openshift.xml like these tests:
https://github.com/jboss-container-images/jboss-kie-modules/pull/504/files#diff-9652da7df90137b93f39a2778369f25a106d0193b55fcef2992a6394a77350c6R10

and here, to grab the result, use xmlint

https://github.com/jboss-container-images/jboss-kie-modules/pull/504/files#diff-9652da7df90137b93f39a2778369f25a106d0193b55fcef2992a6394a77350c6R10